src/odemis/acq/stitching/_registrar.py

```diff
@@ -53,9 +53,9 @@ def __init__(self):
     def addTile(self, tile, dependent_tiles=None):
         """
         Extends grid by one tile.
-        tile (DataArray of shape YX): each image must have at least MD_POS and MD_PIXEL_SIZE metadata. 
+        tile (DataArray of shape YX): each image must have at least MD_POS and MD_PIXEL_SIZE metadata.
         They should all have the same dtype.
-        dependent_tiles (list of DataArray or None): each of the dependent tile, where their position 
+        dependent_tiles (list of DataArray or None): each of the dependent tile, where their position
```

Comment:
What is " each of the dependent tile". I know that you did not write this and only removed the space. However, I think this should be rephrased. Is that a list of the tiles that depend on the "tile" (the param above)? Or the adjacent tiles? What is the "main tile"? The one that is in the tile param? At least it needs to be plural: "each of the dependent tiles"?